virttest/test_setup.py

```diff
@@ -930,7 +930,9 @@ def __init__(self, driver=None, driver_option=None, host_set_flag=None,
                 if "allow_unsafe_assigned_interrupts" in i:
                     self.auai_path = i
         if self.setup:
-            self.sr_iov_setup()
+            if not self.sr_iov_setup():
+                msg = "SR-IOV setup on host failed"
+                logging.error(msg)
```

Comment:
for that I need to use import from autotest.client.shared the "error"  module in test_setup.py. I do not see error.py under avocado/avocado/utils/. Should this be ok?